### DISCUSSION TITLE
如何将樱花token和隧道ID写在配置文件中，而不是命令行-f指定

### DISCUSSION THREAD

**Discussion by 1309896340** [natfrp/wiki#206] [使用问题]
本人自建的mc服务器是Linux操作系统，为了方便启动，将樱花隧道的启动注册成后台服务。由于frpc的命令行启动需要通过-f参数指定token和隧道id，如果要开启多个隧道就需要多写一个配置文件、多配置一个service，非常不方便。
        是否有一种方式，能将这两个-f指定的参数写在配置文件中？

**Comment by fakeboboliu** [comment]
您好，
在隧道列表页面，隧道后面的三个点中有一个“配置文件”项目，
请参考写您自己的配置文件。

**Reply by 1309896340** [reply]
我看过了，但它供我参考的这份配置文件中也没有对隧道ID的配置，仅仅是指定在命令行-f指定的参数中，我怀疑他们没有考虑到这个需求。我是否应该去提议实现这个功能

**Reply by fakeboboliu** [reply]
> 如果要开启多个隧道就需要多写一个配置文件

相关的功能已经实现了，只是还在测试中，并且将与其他更新一起发布，所以现在暂时并不支持。

[a comment by an automated account is not shown]

**Comment by galaxy-eye** [comment]
我觉得可以直接写个脚本开机自启吧
vim /root/frpc.sh
nohup frpc -f xxx > /root/frpc1.txt 2>&1 &
nohup frpc -f xxx > /root/frpc2.txt 2>&1 &

**Comment by Cassy-Lee** [comment]
现在已经支持从配置文件加载了，关闭这个问题@1309896340

**Reply by fakeboboliu** [reply]
感谢您的提醒
